homeassistant/components/ubee/device_tracker.py

```diff
@@ -14,47 +14,59 @@
     CONF_HOST, CONF_PASSWORD, CONF_USERNAME)
 import homeassistant.helpers.config_validation as cv
 
-REQUIREMENTS = ['pyubee==0.2']
+REQUIREMENTS = ['pyubee==0.5']
 
 _LOGGER = logging.getLogger(__name__)
 
+CONF_MODEL = 'model'
+DEFAULT_MODEL = 'detect'
+
+
+def check_model(model):
+    """Check Ubee model."""
+    from pyubee import SUPPORTED_MODELS
```

Comment:
Sure? Anyway, that change with new metadata files

Reply:
Yes, but we're not there yet.

Reply:
Ok, I removed `check_model` function and validation for now. I would really like to avoid specifying list of supported models in this device tracker, because we already have that list in `pyubee` module. If you guys insist, I will add list of supported models in here.

Reply:
Include it in the docs somehow at least.

Reply:
It's already in documentation for [Ubee Device Tracker](https://deploy-preview-9050--home-assistant-docs.netlify.com/components/device_tracker.ubee/).

PR: https://github.com/home-assistant/home-assistant.io/pull/9050